mercury_engine_data_structures/common_types.py

```diff
@@ -4,9 +4,10 @@
 import construct
 from construct import Adapter
 
-from mercury_engine_data_structures.construct_extensions.strings import CStringRobust
+from mercury_engine_data_structures.construct_extensions.strings import CStringRobust, PaddedStringRobust
 
 StrId = CStringRobust("utf-8")
+Char = PaddedStringRobust(1, "utf-8")
```

Comment:
This is kinda defined in `bmsad`, so if you could refactor there.
(1-byte utf-8 is identical to ascii :P)